core/src/services/azfile/backend.rs

```diff
@@ -0,0 +1,426 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use std::collections::HashMap;
+use std::fmt::Debug;
+use std::fmt::Formatter;
+use std::sync::Arc;
+
+use async_trait::async_trait;
+use http::StatusCode;
+use log::debug;
+use reqsign::AzureStorageConfig;
+use reqsign::AzureStorageLoader;
+use reqsign::AzureStorageSigner;
+
+use crate::raw::*;
+use crate::services::azfile::pager::AzfilePager;
+use crate::*;
+
+use super::core::AzfileCore;
+use super::error::parse_error;
+use super::writer::AzfileWriter;
+use super::writer::AzfileWriters;
+
+/// Default endpoint of Azure File services.
+const DEFAULT_AZFILE_ENDPOINT: &str = "file.core.windows.net";
```

Comment:
How about naming as `DEFAULT_AZFILE_ENDPOINT_SUFFIX`? It's not a valid endpoint.

Reply:
solved